charts/console/templates/deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
       {{- end }}
       labels:
         {{- include "console.selectorLabels" . | nindent 8 }}
+        {{ toYaml .Values.podLabels | nindent 8 }}
```

Comment:
Ah, my bad, sorry. I was looking at the `with` version thinking _list_. :man_facepalming:

You were right the first time.

Reply:
My bad as well I didn't really pay attention to your suggestion :)